File: src/Backend/Backend/Models/Dto/EventResourceDto.cs
namespace Backend.Models.Dto
{
    public class EventResourceDto
    {
        public int? ID { get; set; }
        public int SupplierID { get; set; }
        public int EventID { get; set; }
        public int ResourceID { get; set; }
        public int Quantity { get; set; }
        public bool IsReservable { get; set; }
        public EventResourceStatus Status { get; set; }
        public DateTime? StartDateTimeBooked { get; set; }
        public DateTime? EndDateTimeBooked { get; set; }
    }

}

File: src/WebApp/src/app/Models/EventResourceDto.ts
export class EventResourceDto
{
    constructor(
        private id : number,
        private supplierID : number,
        private eventID : number,
        private resourceID : number,
        private quantity : number,
        private isReservable : boolean,
        private status : number,
        private startDateTimeBooked : Date,
        private endDateTimeBooked : Date
    ){}


    getId(): number {
    return this.id;
  }

  getSupplierID(): number {
    return this.supplierID;
  }

  getEventID(): number {
    return this.eventID;
  }

  getResourceID(): number {
    return this.resourceID;
  }

  getQuantity(): number {
    return this.quantity;
  }

  getIsReservable(): boolean {
    return this.isReservable;
  }

  getStatus(): number {
    return this.status;
  }

  getStartDateTimeBooked(): Date {
    return this.startDateTimeBooked;
  }

  getEndDateTimeBooked(): Date {
    return this.endDateTimeBooked;
  }

}